src/openrct2/localisation/Localisation.cpp

```diff
@@ -1223,10 +1223,10 @@ void format_string_to_upper(utf8 *dest, size_t size, rct_string_id format, void
     }
 }
 
-money32 string_to_money(char* string_to_monetise)
+money32 string_to_money(const char* string_to_monetise)
 {
     const char* decimal_char = language_get_string(STR_LOCALE_DECIMAL_POINT);
-    char* text_ptr = string_to_monetise;
+    char* text_ptr = const_cast<char *>(string_to_monetise);
```

Comment:
This removes the correctness of `const` in the argument completely. If you want to use the string as a buffer either copy it, or leave the `const` out of the argument.